lib/roken/getauxval.c

```diff
@@ -213,6 +213,7 @@ rk_getauxval(unsigned long type)
 
     getauxval_sets_errno = 0;
     errno = save_errno;
+    return 0;
 #endif
 #else
```

Comment:
@nicowilliams Did you mean for the code to fall-through and this #else should really be a #endif?

Reply:
can't simply fall though without moving the declaration of a to the top of the function, but that's what it looks like it should do

Reply:
alternate version of this in 366, which i think is right